app/view/set-password.ejs

```diff
@@ -0,0 +1,49 @@
+<% include partials/head.ejs %>
+<title>Set password | Drinkerz</title>
+</head>
+
+<body>
+  <main class="sign-up__container">
+    <section class="sign-up__section">
+      <figure class="sign-up__header">
+        <img src="/static/media/images/logo.png" alt="logo Drinkerz">
+        <h1 class="sign-up__title">Drinkerz</h1>
+      </figure>
+
+      <form class="sign-up__form" action="/set-password" method="POST">
+        <p>
+          <%- user.username %>
+        </p>
+
+        <label>Password</label>
+        <input type="password" name="password" placeholder="password">
+
+        <label>Gender</label>
+        <input type="radio" name="gender" value="male"> Male<br>
+        <input type="radio" name="gender" value="female"> Female<br>
+        <input type="radio" name="gender" value="other"> Other
+
+        <label>Age</label>
+        <input type="text" name="age" placeholder="age">
+
+        <p>preferred matches</p>
+
+        <fieldset class="preferred__age">
+          <label>Set age range</label>
+          <input type="text" name="age_min" placeholder="20">
```

Comment:
Make the min age 18 instead of 20, you can drink beer if you're 18 ;)

Reply:
Noice